src/channels/__init__.py

```diff
@@ -507,16 +518,18 @@ def incEventValue(eventId: int, step: int, res: float) -> int:
 
 
 def getRecEventId(index: int) -> int:
-    """Returns recording event ID for channel at `index`.
-    
+    """Return the starting point of REC event IDs for the channel at `index`.
+
+    See `general.processRECEvent` for more information.
+
     HELP WANTED: Honestly REC events are sooooo confusing, and I avoid using 
     them entirely. Can someone else explain them?
```

Comment:
If you feel like it's explained sufficiently, feel free to get rid of the `HELP WANTED` flags

Reply:
We definitely need to write proper examples and tutorials regarding this topic, but in the scope of docstrings, my explanation is probably sufficient for now.